barretenberg/cpp/src/barretenberg/stdlib/protogalaxy_verifier/recursive_decider_verification_key.hpp

```diff
@@ -101,7 +102,9 @@ template <IsRecursiveFlavor Flavor> class RecursiveDeciderVerificationKey_ {
     {
         auto native_honk_vk = std::make_shared<NativeVerificationKey>(verification_key->circuit_size,
                                                                       verification_key->num_public_inputs);
-        native_honk_vk->pcs_verification_key = verification_key->pcs_verification_key;
```

Comment:
odd this worked before...

Reply:
yeah it was indeed setting it to null in the previous usage but we just never ended up using it. It only came up for me in the verification of the AztecIvc proof